pkg/bootstrap/static.go

```diff
@@ -0,0 +1,323 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bootstrap
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
We use template to generate bootstrap - please don't add dependencies to control plane, proto  in bootstrap !

This is not a static config by any means !

Reply:
Static config does not mean it's not type checked and validated.

Reply:
It's a static file - you can add any offline presubmit validation you want. 

We are not going back to that design.